Content.Server/Atmos/Piping/Unary/EntitySystems/GasVentPumpSystem.cs

```diff
@@ -80,10 +89,15 @@ private void OnGasVentPumpUpdated(EntityUid uid, GasVentPumpComponent vent, ref
                 return;
             }
 
+            if (vent.IsPressureLockoutManuallyDisabled && (_timing.CurTime - vent.ManualLockoutDisabledAt).TotalSeconds > vent.ManualLockoutDisabledDuration)
```

Comment:
Don't save the time the lockout was disabled, save the time it should be enabled again as a TimeSpan instead. That way you can directly compare with CurTime and don't have to subtract it every single update.